shell/packages/sandstorm-ui-app-details/app-details.html

```diff
@@ -52,7 +52,7 @@
   --}}
   <div class="app-details-widget">
     {{debug}}
-    <div class="app-icon" style="background-image: url('{{appIconSrc}}');"></div>
+    <div class="app-icon" style="background-image: url('{{appIconSrc}}'); height: 128; width: 128;"></div>
```

Comment:
This width and height you added are invalid because they don't specify any units, so the browser ignores them. I removed them in 9a9b058.